pkg/Microsoft.Private.PackageBaseline/packageIndex.json

```diff
@@ -3830,11 +3830,13 @@
       "BaselineVersion": "4.3.0",
       "InboxOn": {
         "netcoreapp2.0": "4.0.2.0",
-        "netcoreapp2.1": "4.0.3.0"
+        "netcoreapp2.1": "4.0.3.0",
+        "netcoreapp3.0": "4.1.0.0"
```

Comment:
@ericstj this is one thing I was not sure about - should it be `netcoreapp3.0`?

Reply:
3.0 is correct once https://github.com/dotnet/corefx/pull/32014 is merged.  I believe it is safe to leave this at 3.0 and will save @weshaggard some work.